cfgov/v1/tests/atomic_elements/organisms/test_organisms.py

```diff
@@ -300,7 +300,9 @@ def test_render(self):
         ):
             self.assertIn(
                 f'<a class="m-list_link"\n                       '
-                f'href="{url}"\n                       >{text}</a>',
+                f'href="{url}"\n                       >\n'
+                f"                       {text}\n"
+                f"                    </a>",
```

Comment:
Unrelated to your changes, this test and the others like it in this file bother me. I'm not certain why we're testing that this renders with specific whitespace.

Reply:
Yeah it was a huge pain to get that test to resolve accurately due to the arbitrary-ish whitespace. It works for now but is probably worth revisiting?

Reply:
I think if we revisit, we should revisit that entire file. This isn't the only test like that. So, no action needed for this PR.